tasks/main.yml

```diff
@@ -191,8 +191,8 @@
       - restart nfs
 
   - name: Copy NFS export conf file
-    copy:
-      src: ../files/nfs-exports
+    template:
```

Comment:
Revert this block to the prior setup
add condition `when: not secure_nfs`

Add a new block exactly like this, and add `when secure_nfs`

Add to the base vars.yaml